pkg/constants/constants.go

```diff
@@ -34,11 +34,15 @@ const (
 	// ModelMountPath is the volumeMount path for model.
 	ModelMountPath string = "/workspace/model"
 
-	// JobTrainerNode is the Job name for the trainer node.
-	JobTrainerNode string = "trainer-node"
+	// AncestorTrainer is the ancestor name for Trainer, which is mostly used for the value of
+	// 'trainer.kubeflow.org/trainjob-ancestor-step'.
+	AncestorTrainer string = "trainer"
 
-	// ContainerTrainer is the container name for the trainer.
-	ContainerTrainer string = "trainer"
+	// JobNode is the Job name for the trainer node.
+	JobNode string = "node"
+
+	// ContainerNode is the container name for the node.
+	ContainerNode string = "node"
```

Comment:
Similar to initializer constants, shell we keep single value to reduce number of constants we maintain ?
https://github.com/kubeflow/trainer/blob/ac6bc1f098642529031dcbb1e4fb0d774a52a976/pkg/constants/constants.go#L32
```suggestion
	// Node is the name of the Job and container for the trainer node
	Node string = "node"
```

Reply:
Sure.